app/views/categories/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1><%= category.name %></h1>
+
+<p><%= t(".page_description", name: category.name) %></p>
+
+<p><%= link_to t(".navigation.helpers.back"), categories_path %>
+
+<% posts.each do |post| %>
+  <%= render "posts/blog_post", post: post %>
+<% end %>
```

Comment:
If you name your partial `_post.html.erb` you should be able to replace the `#each` loop with:

```erb
<%= render posts %>
```